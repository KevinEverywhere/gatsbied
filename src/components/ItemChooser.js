import React from 'react'
import { Link } from 'gatsby'

const ItemChooser = (props) => (
  <li><Link to={props.item.toLowerCase()}>{props.item}</Link></li>
)

export default ItemChooser
