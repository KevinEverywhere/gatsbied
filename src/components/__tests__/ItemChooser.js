import React from "react"
import renderer from "react-test-renderer"
import ItemChooser from "../ItemChooser"

describe("ItemChooser", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<ItemChooser item="Work" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
