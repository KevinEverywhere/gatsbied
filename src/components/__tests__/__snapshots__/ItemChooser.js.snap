// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemChooser renders correctly 1`] = `
<li>
  <a
    href="work"
  >
    Work
  </a>
</li>
`;
